src/controllers/paper.filing.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../types/template.paths";
+import { URL_QUERY_PARAM } from "../types/page.urls";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { getCompanyProfile } from "../services/company.profile.service";
+import { isCompanyNumberValid } from "../validators/company.number.validator";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const companyNumber = req.query[URL_QUERY_PARAM.COMPANY_NUM] as string;
+    if (!isCompanyNumberValid(companyNumber)) {
+      return next(new Error("Invalid company number entered in invalid.company.status url query parameter"));
```

Comment:
the error has the incorrect controller name in the message. Could do as martin suggest on my PR and replace the controller name with the paper filing path (but don't substitute the company number as we shouldn't log user data incase it is malicious causing log injection attack)